pam/model.go

```diff
@@ -137,20 +139,8 @@ func (m *model) Update(msg tea.Msg) (tea.Model, tea.Cmd) {
 		m.brokerSelectionModel.SetWidth(m.width)
 
 	// Exit cases
-	case pamIgnore:
-		m.exitMsg = msg
-		return m, m.quit()
-	case pamAbort:
-		m.exitMsg = msg
-		return m, m.quit()
-	case pamSystemError:
-		m.exitMsg = msg
-		return m, m.quit()
-	case pamAuthError:
-		m.exitMsg = msg
-		return m, m.quit()
-	case pamSuccess:
-		m.exitMsg = msg
+	case pamReturnStatus:
```

Comment:
I think we can achieve the same thing with testing if the returned value implements `pamReturnStatus` interface without returning the interface with an assertion directly if you want to minimize the switch cases.

Reply:
Both ways are fine to me.